integration/build.gradle

```diff
@@ -29,9 +29,9 @@ dependencies {
         googleNotebook = "v1-rev20201110-1.30.10"
 
         datarepoClient = "1.0.155-SNAPSHOT"
-        testRunnerVersion = "0.1.3-SNAPSHOT"
+        testRunnerVersion = "0.1.4-SNAPSHOT"
```

Comment:
I was getting a ClassDefNotFound error from test runner code (after upgrading terra-common-lib version), and this fixed it.